paddle/fluid/fluid_document.spec

```diff
@@ -0,0 +1,376 @@
+ ('paddle.fluid.nets.img_conv_group', '3802be78fbfb206dae64a2d9f8480970')
+ ('paddle.fluid.optimizer.SGD', 'e2cbf4cc27e11cfea3abc87c4d07d31b')
+ ('paddle.fluid.optimizer.Momentum', '072d95b0c1c8030e767d3de941148724')
+ ('paddle.fluid.optimizer.Adagrad', 'e166e1bfab50ce5e55270a9071b9748c')
+ ('paddle.fluid.optimizer.Adam', '9d6bef9137712a2faa777175c7c14b5f')
+ ('paddle.fluid.optimizer.Adamax', '2820d451b0c06528b71883f78159eacf')
+ ('paddle.fluid.optimizer.DecayedAdagrad', '778d692c84f3fb129d2e20b517e6f54f')
+ ('paddle.fluid.optimizer.Ftrl', '1fd5048f2fb6e6fb126e2e58b2a98467')
+ ('paddle.fluid.optimizer.SGDOptimizer', 'e2cbf4cc27e11cfea3abc87c4d07d31b')
+ ('paddle.fluid.optimizer.MomentumOptimizer', '072d95b0c1c8030e767d3de941148724')
+ ('paddle.fluid.optimizer.AdagradOptimizer', 'e166e1bfab50ce5e55270a9071b9748c')
+ ('paddle.fluid.optimizer.AdamOptimizer', '9d6bef9137712a2faa777175c7c14b5f')
+ ('paddle.fluid.optimizer.AdamaxOptimizer', '2820d451b0c06528b71883f78159eacf')
+ ('paddle.fluid.optimizer.DecayedAdagradOptimizer', '778d692c84f3fb129d2e20b517e6f54f')
+ ('paddle.fluid.optimizer.RMSPropOptimizer', '1ff43847fdb81d603ce48cc83a804fdb')
+ ('paddle.fluid.optimizer.FtrlOptimizer', '1fd5048f2fb6e6fb126e2e58b2a98467')
+ ('paddle.fluid.optimizer.Adadelta', '8a7aa2d654796ef3c7c578bc846efe26')
+ ('paddle.fluid.optimizer.ModelAverage', 'a0ce38eb9c92cf1eae28722b66a94bd2')
+ ('paddle.fluid.optimizer.LarsMomentum', '0be7e17e374bc11a0c417f58ddae3695')
+ ('paddle.fluid.optimizer.LarsMomentumOptimizer', '0be7e17e374bc11a0c417f58ddae3695')
+ ('paddle.fluid.profiler.cuda_profiler', '2e2fb1cfc469a67f19fb578a2ed6be79')
+ ('paddle.fluid.profiler.reset_profiler', '397ce757fabbe5c622e0c3458c41fcd0')
+ ('paddle.fluid.profiler.profiler', 'bd3a07eeb68e384f4d2d416cb2e28d86')
+ ('paddle.fluid.profiler.start_profiler', '88da8fb6dbebaee2f7520188a09574f9')
+ ('paddle.fluid.profiler.stop_profiler', 'a7500e39dd033f1e64f562e909333a8a')
+ ('paddle.fluid.recordio_writer.convert_reader_to_recordio_file', '65c7523e86f0c50bb729b01667f36310')
+ ('paddle.fluid.recordio_writer.convert_reader_to_recordio_files', 'bc643f0f5f1b9db57ff0d8a57d379bd7')
+ ('paddle.fluid.regularizer.L1Decay', '5fa1107d292765dc06fe81b7612d6ea2')
+ ('paddle.fluid.regularizer.L2Decay', 'f83c7a9862940772424eb0ae79ca21d7')
+ ('paddle.fluid.regularizer.L1DecayRegularizer', '5fa1107d292765dc06fe81b7612d6ea2')
+ ('paddle.fluid.regularizer.L2DecayRegularizer', 'f83c7a9862940772424eb0ae79ca21d7')
+ ('paddle.fluid.transpiler.DistributeTranspiler', '8653d0a1568d1524ab2e664cc443f8b1')
+ ('paddle.fluid.transpiler.memory_optimize', 'eda17d0f1639bc6ca215cecf87f588a4')
+ ('paddle.fluid.transpiler.release_memory', 'ac4114d3df16264f1946deb3a8434a6f')
+ ('paddle.fluid.transpiler.HashName', 'a519f5335c70bd17e467d5d9af559bc1')
+ ('paddle.fluid.transpiler.RoundRobin', '7b22b6f83d9f3f782f58df18933dc7cb')
+ ('paddle.fluid.transpiler.DistributeTranspilerConfig', 'cb46bd12b047faf5eeb405ff60f01b0a')])
```

Comment:
I'm thinking a better way to not to add `.spec` files in the reponsitory, e.g. we can generate the develop spec file before applying current PR changes, then generate again the changed spec after the PR branch is applied. So that developers won't care about to update this file if anything was going to be changed.

The reason for the API.spec is that it can be generated only when the branch was built. `.spec` file can reduce build time on CI environment. But for this change seems we can apply the method said above.

Reply:
@panyx0718 please take a look at this? seems this file still need to build and generate `whl` package first before checking.